styleguide/_themes/derek/scss/components/_footer.scss

```diff
@@ -1 +1,87 @@
-
+a.footer-menu-link {
```

Comment:
@runrog this is almost there. Please remove the element prefixes here and elsewhere in this file. A class definition does not need an element prefix.